### DISCUSSION TITLE
a point system like metis

### DISCUSSION THREAD

**Discussion by iman136938** [teaxyz/white-paper#141] [Ideas]
i suggest the tea protocol team implement a xp system like currently running metis test net (weekly and every week different quests ) for the upcoming testnet

**Comment by Amplitude88** [comment]
Would be nice. I like the way metis is going because they have the engagement. Unfortunately i think we will have some problems by sybils also. :-/

**Reply by rav2525** [reply]
hope the system gets better

**Comment by hanriiver** [comment]
how can i get contribute

**Comment by etherbiln** [comment]
Not ,I think its best for the community 😊
